mkat_fpga_tests/test_cbf.py

```diff
@@ -701,6 +701,9 @@ def test_vacc(self):
         q_denorm = 128
         quantiser_spectrum = get_quant_snapshot(
             self.correlator, test_input) * q_denorm
+
```

Comment:
Seeing as how this is still on the vacc-test-aqfizing branch, I think you should change the assertTrue's to be Aqf.is_true...